components/ListItemSquare/index.js

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+import Image from "next/image";
```

Comment:
As I mentioned in[ my answer of your question](https://github.com/orgs/neuefische/discussions/320#discussioncomment-4166847), this could be a solution of your mean warnings  
```suggestion
import Image from "next/legacy/image";
```